exercises/anagram/src/Example.hx

```diff
@@ -0,0 +1,17 @@
+package;
+
+class Anagram {
+    public static function findAnagrams(subject: String, candidates: Array<String>): Array<String> {
+        var anagrams = [for (val in candidates) 
+                        if (subject.toLowerCase() != val.toLowerCase())
+                        if (sort_string(subject.toLowerCase()) == sort_string(val.toLowerCase()))
+                        val];
+        return anagrams;
+    }
+    
+    private static function sort_string(string:String): String {
```

Comment:
Update sort_string to sortString and add a space between string:String for consistency.